Dockerfile

```diff
@@ -6,7 +6,7 @@
 # Please note that it'll use about 1.2 GB disk space and about 15 minutes to
 # build this image, it depends on your hardware.
 
-FROM ubuntu:16.04
+FROM ubuntu:xenial-20210114
```

Comment:
Well, I really prefer not to specify such precise version tag as it'll only make it obsolete more frequently.

Reply:
what would you suggest?

Reply:
I prefer to stay the way that we just specify the LTS version of it ;)

Reply:
Here comes #2540, as I said, it'll only make it obsolete more frequently.

#2537 is using xenial-20210114 and #2540 is using xenial-20210416, it's more like keeping using the latest snapshot tag...

Reply:
What tag can we use that would avoid this?

Reply:
I believe the original should be good enough.

Reply:
Actually dependabot or renovate seem to do a better job for the familiar feature, like not to send duplicated pull request for the change on a same thing.

Reply:
The original doesn't seem to update in the manner you're implying, or else snyk wouldn't be sending these PRs.

Reply:
Oh I mean the version in the master branch is the "the original", not the version in the pull request.

If Snyk already sent a PR to update ubuntu to `xenial-20210114` and it found that there is a new version `xenial-20210416`, I think it should just update the previous PR instead of sending a new one. Anyway, it's a little off topic.

Reply:
yes, that's what i'm saying tho - if the master branch's tag auto-updated, then none of the snyk PRs would be needed.

(i totally agree that snyk should have updated the first instead of opening the second, but that's indeed a separate topic)

Reply:
I'm not sure what "if the master branch's tag auto-updated" means? When `xenial-20210114` was tagged, I believe `xenial` and `16.04` will also be updated, just not specified to be a certain snapshot like `xenial-20210114`, but that'll be good enough for me.

Reply:
hmm. then that implies that snyk shouldn't be warning on this at all. let me ask them.

Reply:
Hi folks 👋

I shared with the team internally at Snyk.
Looks like there's some planned work for this on Q4 although I'm not sure to which detail/extent. Might be to update/rebase a PR, or to manage the noise level down.

Reply:
Cool, thanks @lirantal !

Reply:
Nothing new on my end yet, but feel free to reach out with a ping here or https://twitter.com/liran_tal if you wanted to chime in on anything.